autoload/metarw.vim

```diff
@@ -98,6 +98,10 @@ endfunction
 
 function! metarw#_event_handler(event_name)  "{{{2
   let fakepath = expand('<afile>')
+  if filereadable(fakepath)
+    execute '%!cat ' . fakepath
```

Comment:
I believe this is not the right way to fix.  `metarw#_event_handler` is called by various events, not only for `:e some:existing_file`. This diff causes unexpected side effect for other commands, for example, `:w some:existing_file`, `:r some:existing_file`, etc.

Reply:
thanks.
I'll fix it in my spare time.